.luacheckrc

```diff
@@ -23,13 +23,14 @@ read_globals = {
     "mouse",
     "mousegrabber",
     "root",
-    "screen",
     "selection",
     "tag",
     "window",
 }
+
 -- May not be read-only due to client.focus
 globals = {
+    "screen",
```

Comment:
Well, because the new version of luacheck complain if you do `screen[1].foo = "bar"` while `local s = screen[1]; s.foo = "bar"` still work. I will add it to the commit message.